src/commandsAndMenu.ts

```diff
@@ -190,6 +192,21 @@ export function addCommands(
     }
   });
 
+  /** Add git open gitignore command */
+  commands.addCommand(CommandIDs.gitOpenGitignore, {
+    label: 'Open .gitignore',
+    caption: 'Open .gitignore',
+    isEnabled: () => model.pathRepository !== null,
+    execute: async () => {
+      model.ensureGitignore();
```

Comment:
```suggestion
      await model.ensureGitignore();
```

Reply:
Thx for the review. Last commit implements this.